src/platforms/hosted/windows/ftdi.h

```diff
@@ -1,11 +1,8 @@
 /*
  * This file is part of the Black Magic Debug project.
  *
- * Copyright (C) 2011  Black Sphere Technologies Ltd.
- * Copyright (C) 2018  Uwe Bonnes(bon@elektron.ikp.physik.tu-darmstadt.de)
- * Written by Gareth McMullin <gareth@blacksphere.co.nz>
  * Copyright (C) 2022-2023 1BitSquared <info@1bitsquared.com>
```

Comment:
Given this file didn't exist till this year, the `2022-` clause should be dropped. Given the next line, are you wishing to keep copyright or did you mean `Written by`? We don't mind either way, just want to be sure.